Description: Fix test failure that uses env from rapid plugins
 The test main.datadir_permission has a custom config that tries to load the
 env variable MYSQLXPLUGIN_DIR. In 5.7.14 most of the mysqlx related data was
 moved to the rapid plugin system, and may not be available during the test
 run. Since the test doesn't seem to actually need the custom cnf, we simply
 delete it until the issue is fixed upstream.
Author: Lars Tangvald <lars.tangvald@oracle.com>
Bug: http://bugs.mysql.com/bug.php?id=83321
Forwarded: not-needed
Last-Update: 2016-10-11

--- mysql-5.7-5.7.15.orig/mysql-test/t/datadir_permission.cnf
+++ /dev/null
@@ -1,5 +0,0 @@
-# Use default setting for mysqld processes
-!include include/default_my.cnf
-
-[mysqld.1]
-plugin-dir=@env.MYSQLXPLUGIN_DIR
